epoxy-sample/src/main/java/com/airbnb/epoxy/sample/models/ColorModel.java

```diff
@@ -54,6 +61,31 @@ public void bindChanges(@NonNull ColorHolder holder,
     }
   }
 
+  @Nullable
+  @Override
+  public EpoxyItemAnimation bindChangesAnimated(@NonNull ColorHolder view,
+                                                @NonNull EpoxyModel<?> previouslyBoundModel,
+                                                @Nullable EpoxyItemAnimation interruptedAnimation) {
+
+    int currentColor = ((ColorModel) previouslyBoundModel).color;
+    float currentRotation = 0f;
+
+    if(interruptedAnimation != null) {
+      AnimatorSet previousAnim = (AnimatorSet) ((EpoxyItemAnimation.AnimatorWrapper) interruptedAnimation).wrappedAnimator;
```

Comment:
Unfortunately, the `EpoxyItemAnimation` interface is needed so you can use animations that are not `Animator`s, for example `ViewPropertyAnimatorCompat`, Lottie, etc...

I guess you don't really need the previous animation and could get the values directly from the holder, the same is probably true for `previousModel` as well.

I've been thinking, do we actually need all that stuff? What would happen if we completely ignored all RV conventions and did all the binding in `ItemAnimator.animateChange` or analogously all the animations in `Adapter.bindViewHolder`? The user could hold on to the previous animation in the model and stop all animations in `unbind`.